CHANGELOG.md

```diff
@@ -104,6 +104,8 @@ swallowed. The events are sent through the pipeline.
 - Include zero-valued integers in JSON output for all types.
 - Check event entities now have a last_seen timestamp.
 - Improved silenced entry display and UX.
+- Add version output for dev and nightly builds (#1320).
+- Improve git version detection by directly querying for the most recent tag.
```

Comment:
Since we just made a release, maybe try to merge master back into your branch, just to make sure these lines don't get appended to an already released version!

Reply:
Agreed, I've rebased off of master and updated the changelog.